packages/readabilityjs/Readability.js

```diff
@@ -2774,17 +2784,30 @@ Readability.prototype = {
           // some website like https://substack.com might have their custom styling of tweets
           // we should omit ignoring their particular case by checking against "tweet" classname
           (weight >= 25 && linkDensity > 0.5 && !(node.className === "tweet" && linkDensity === 1)) ||
-          ((embedCount === 1 && contentLength < 75) || embedCount > 1);
+          ((embedCount === 1 && contentLength < 75) || embedCount > 1))
 
-        if (haveToRemove)
-          this.log("Cleaning Conditionally", { node, className: node.className, children: Array.from(node.children).map(ch => ch.tagName) });
+        if (haveToRemove) {
+          this.log("Cleaning Conditionally", { className: node.className, children: Array.from(node.children).map(ch => ch.tagName) });
+        }
 
         return haveToRemove;
       }
       return false;
     });
   },
 
+  _isOmnivoreNode: function(node) {
+    var walk = node
+
+    while (walk) {
+      if (walk.className && node.className.startsWith("_omnivore")) {
```

Comment:
Is the `node` here supposed to be `walk`?

Reply:
😬  yes!